reports/default/examples/full.json

```diff
@@ -906,7 +906,8 @@
                         "end": "2015-07-07 00:00:00"
                     },
                     "restrictions": {
-                        "type": "WITHOUT RESTRICTIONS"
+                        "type": "WITHOUT RESTRICTIONS",
```

Comment:
Change `type` here for `WITH RESTRICTIONS`.

Reply:
Done